src/components/Nav/Nav.js

```diff
@@ -1,7 +1,87 @@
-import React from 'react';
+import { AiOutlineMenu } from 'react-icons/ai';
+import { CgSearch } from 'react-icons/cg';
+import styled from 'styled-components';
+import React, { useState } from 'react';
+import SideDrawer from './SideDrawer';
+import GlobalFonts from '../../fonts/fonts';
 
 const Nav = () => {
-  return <div>Nav</div>;
+  const [navDrawer, setNavDrawer] = useState(false);
+
+  const handleNavDrawer = () => {
+    setNavDrawer(!navDrawer);
+  };
+
+  return (
+    <>
+      <GlobalFonts />
+      <SideDrawer navDrawer={navDrawer} closeDrawer={handleNavDrawer} />
+      <Navigator>
+        <NavWrapper>
+          <NavElement>
+            <NavBtn onClick={handleNavDrawer}>
+              <AiOutlineMenu className="navIcon" />
```

Comment:
- styled-component를 이용해서 스타일링이 된 태그들은 해당 definition으로 이동해서 바로 스타일링을 확인이 가능한데 className으로 선언한 부분은 바로 definition으로 이동이 불가능해서 스타일링 파악하기에 어려움이 있습니다.
- 위에 내용은 참고하시면 좋고 아이콘의 사이즈 조절을 위해 className을 사용하신 코드는 아래와 같은 경우로도 사이즈 조절이 가능합니다.
```suggestion
              <AiOutlineMenu size={20} />
```